src/components/screens/InfoCard.js

```diff
@@ -71,8 +69,17 @@ export const InfoCard = ({ addresses, category, contact, contacts, webUrls, open
   <View>
     {!!category && !!category.name && (
       <InfoBox>
-        <RNEIcon name="list" type="material" color={colors.primary} iconStyle={styles.margin} />
-        <RegularText>{category.name}</RegularText>
+        <RNEIcon
+          accessible={true}
+          accessibilityLabel="Menü icon"
+          name="list"
+          type="material"
+          color={colors.primary}
+          iconStyle={styles.margin}
+        />
+        <RegularText accessible={true} accessibilityLabel={`(Kategorie) (${category.name}`}>
```

Comment:
here you started a `(` but have not closed it `(${category.name}`.
as there is no text after it and the text before is already wrapped, the category name does not need to be wrapped in `(...)`